x/emissions/module/rewards/rewards.go

```diff
@@ -35,7 +37,8 @@ func EmitRewards(ctx sdk.Context, k keeper.Keeper, blockHeight BlockHeight, weig
 
 	totalRewardToStakedReputers := alloraMath.ZeroDec()
 	// for every topic
-	for topicId, topicReward := range topicRewards {
+	for _, topicId := range sortedTopics {
```

Comment:
shouldnt this iterate through topicRewards? if not, why not? 
(ie does `GenerateRewards...` guarantee same keyset?)

Reply:
`GenerateRewards` should only ever return a strict subset of the values in `sortedTopics` but I also agree, there's no reason to write it this way when we can simply sort topicRewards

Reply:
ah I see using sortedTopics reduces the need to sort twice

but it also looks like someone's merged PR overwrote my PR to remove the non-generic sort functions

Reply:
a strict subset is not the same as the same subset, is that situation covered?